src/components/GameLobby.js

```diff
@@ -56,28 +46,28 @@ function GameLobby({ userName, gamesWon, _id }) {
       setRoom(localStorageRoom);
     }
 
-
-    // Receives available rooms & players from back end and sets it to useState
-    // rooms: [{room: 1,
-    //        players: ["name1", "name2", "name3"]
-    //     },
-    //     {room: 2,
-    //       players: ["name4", "name5", "name6"]
-    //    }]
     socket.on("available_rooms", (rooms) => {
       setAvailableRooms(rooms);
     });
   }, [socket]);
 
+  // TEMP background image - liable to change or be moved into App.js, across the entire app
+  const backgroundImage = {
+    backgroundImage: `url(${require('/Users/cesar/code-the-dream/practicum/bb-practicum-team-2-front/src/img/forest-bg.jpg')})`,
```

Comment:
Where is this location coming from? Where is **/Users** path? this needs to be either in the **public** folder or in the **src** folder.

Reply:
Whoops, I used the relative path by accident, I just updated it to the proper path.